### PR TITLE
Feature/ccv2 common branch

### PR DESCRIPTION
**Pull request by sijo5722-2021**

### CODE REVIEW

File: mojaloop/iac/playbooks/argoeks_cluster_deploy.yaml [hunk 1]
```diff
@@ -2,12 +2,16 @@
   become: true
   roles:
     - mojaloop.iac.bastion_common
-    - mojaloop.iac.netclient
-    - mojaloop.iac.haproxy
+    - mojaloop.iac.netbird_client
+    #- mojaloop.iac.eks_cleanup
+
+- hosts: "{{ groups['bastion'] | first }}"
```
Comment (dfry) [inline mojaloop/iac/playbooks/argoeks_cluster_deploy.yaml:8]: add sort filter here as well please @jmutai

**Review by dfry — CHANGES_REQUESTED** [review]
please address adding sort

File: mojaloop/iac/playbooks/argoeks_cluster_deploy.yaml (hunk 1 above)
Comment (jmutai) [reply mojaloop/iac/playbooks/argoeks_cluster_deploy.yaml]: update done.

**Review by dfry — APPROVED** [review]
lgtm

**Pull request merged**